### PR TITLE
Remove lodash/fp from personalization

### PR DESCRIPTION
**Pull request by theodur**
## Description
Remove `lodash/fp` from the `personalization` directory.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#1678

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/dashboard/reducers/folders.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import set from 'lodash/fp/set';
+import set from 'platform/utilities/data/set';
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/reducers/folders.js:2]: How did I not know this was a drop-in replacement?! 🤯 

Also, could you prefix all  the new platform imports with `~/`, so `~/platform/utilities/data/set`?

File: src/applications/personalization/dashboard/reducers/prescriptions.js [hunk 2]
```diff
@@ -69,7 +68,7 @@ export default function prescriptions(state = initialState, action) {
         };
       }
 
-      return assign(state, newState);
+      return { ...state, ...newState };
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/reducers/prescriptions.js:71]: One of these days I'm going to try using Immer in our reducers like the Redux maintainers have been recommending for ages.

**Review by erikphansen — CHANGES_REQUESTED** [review]
could you update the platform imports to use `~/`?

File: src/applications/personalization/dashboard/reducers/folders.js (hunk 1 above)
Comment (theodur) [reply src/applications/personalization/dashboard/reducers/folders.js:2]: Yup! As long as currying isn't involved, this function can be swapped in as a replacement!

And yeah sure, I can add that to the imports. Out of curiosity, is it a matter of consistency/preference? I noticed most of the platform imports in the `personalization` directory have `~/` prepended to them.

Comment (erikphansen) [reply src/applications/personalization/dashboard/reducers/folders.js:2]: Yes just a preference and to ensure that no one (like a new dev) thinks that `platform` is from node_modules

Comment (theodur) [reply src/applications/personalization/dashboard/reducers/folders.js]: Yeah that makes sense. The imports should be good now ✅

**Review by erikphansen — APPROVED** [review]

**Pull request merged**